lib/phaxio/client.rb

```diff
@@ -93,6 +90,11 @@ def handle_response response
         end
       end
 
+      def tmpname(extension)
+      	t = Time.now.strftime("%Y%m%d")
```

Comment:
Tabs?

![kill it with fire](https://thumbs.gfycat.com/DevotedPleasingBighorn-size_restricted.gif)